packages/astro/src/core/errors/errors-data.ts

```diff
@@ -1808,6 +1808,18 @@ export const ActionCalledFromServerError = {
 	hint: 'See the `Astro.callAction()` reference for usage examples: https://docs.astro.build/en/reference/api-reference/#callaction',
 } satisfies ErrorData;
 
+/**
+ * @docs
+ * @description
+ * Cannot the module without enabling the experimental feature
+ */
+export const CantUseManifestModule = {
+	name: 'CantUseManifestModule',
+	title: 'Cannot the module without enabling the experimental feature.',
```

Comment:
```suggestion
 * Cannot use the module without enabling the experimental feature.
 */
export const CantUseManifestModule = {
	name: 'CantUseManifestModule',
	title: 'Cannot use the module without enabling the experimental feature.',
```
I'm assuming this is future proof for different modules, so we're not saying *which* module can't be used? (Otherwise, I'd normally suggest e.g. "the `astro:config` module" (or "an `astro:config` module") if that's the only possibility. And I don't think it's a problem to not specify the sub-path here? I think just saying `astro:config` here is fine and better than nothing if you only mean `astro:config/client` and `astro:config/server`?)

Reply:
No, this is an error created ad-hoc for `astro:config`. It's not future proof. I updated your suggestion. Is that good?

Reply:
Yup, looks good to me! Thoughts about making the same change on line 1814, too?

Reply:
Added :)